src/content/advanced/gitops/apps/add_app_template.md

```diff
@@ -0,0 +1,99 @@
+---
+linkTitle: Add a new App Template
+title: Add a new App Template
+description: Learn how to create application templates to deploy apps using GitOps.
+weight: 50
+menu:
+  main:
+    identifier: advanced-gitops-apps-add-app-template
+    parent: advanced-gitops-apps
+last_review_date: 2023-02-10
+user_questions:
+  - How can I create an template for app deployment in GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
```

Comment:
I was thinking if we can add an intro in each of these 'Child' entries so in case someone comes from Google she knows that is part of a whole thing. WDYT?

Reply:
added a small paragraph with a link to platform overview